testsuite/integration/microprofile-tck/fault-tolerance/pom.xml

```diff
@@ -100,6 +100,7 @@
                         <!-- Increase timeout to reduce timing intermittent failures -->
                         <org.eclipse.microprofile.fault.tolerance.tck.timeout.multiplier>2.0</org.eclipse.microprofile.fault.tolerance.tck.timeout.multiplier>
                     </systemPropertyVariables>
+                    <runOrder>reversealphabetical</runOrder>
```

Comment:
Any particular reason for this?

Reply:
MetricsDisabledTest and AllMetricsTest appear to be order-dependent, but not by design. It seems the same smallrye-metrics change that broke some tests for that spec upgrade is at work here. I made a change to our tests suggested by Ladislav Thon (https://github.com/smallrye/smallrye-fault-tolerance/blob/master/testsuite/tck/src/test/java/io/smallrye/faulttolerance/tck/CleanupMetricRegistries.java), but that failed to fix the issue. While this did, it's hardly ideal. I'm happy enough to stick with it for now if need be, but if you, being a bit more familiar with this part of the system, can find a better fix, I'd love to see it. :)

Reply:
Thanks, I will look into it.